docker/dvc_operator.Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.6.7-slim
+
+ENV PYTHONUNBUFFERED 1
+
+RUN apt-get update --fix-missing --no-install-recommends && \
+    apt-get upgrade -y && \
+    apt-get install -y build-essential \
+    python-dev git \
+    libffi-dev libssl-dev libpq-dev \
+    wget && \
+    pip install -U --no-cache-dir pip setuptools wheel dvc && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* && \
+    rm -rf /tmp/*
+
+COPY run_poodle.sh /run_poodle.sh
+
+RUN mkdir /src && cd /src && \
+	git clone https://github.com/yuxiangl/poodle-dvc.git && \
```

Comment:
In prod, we'll need to make sure the machine that runs this has access to the repo. That could complicate things.

Alternatively, we auth with PyPI and `pip install` the right version of the `bidding` package (I already know how to do that, and it's set up for that in prod).